### PR TITLE
JavaScript and CSS progressive enhancement

### PR DESCRIPTION
**Pull request by regiskuckaertz**
In this PR, we

- start using the FT polyfills service (our own hosted version)
- sign the root element with feature-detected classes
- manually polyfill `URLSearchParams`

cc @janua

### CODE REVIEW

**Comment by janua** [comment]
:clap: 
Just making some changes to the polyfill service now!

**Comment by regiskuckaertz** [comment]
🏇 

Is there a way to know when they add new polyfills? I need to polyfill `URLSearchParams` myself, not a big deal but one to add to the list when it is available 🌞

**Comment by janua** [comment]
I guess you could PR the polyfill service to add the ones you would like! :smile:

**Comment by regiskuckaertz** [comment]
@janua oooh, that's a good idea. Will do 😃

File: src/_shared/js/capi-multiple.js [hunk 1]
```diff
@@ -1,189 +1,190 @@
 import { getIframeId, getWebfonts, resizeIframeHeight, reportClicks } from
-	'./messages.js';
+    './messages.js';
 import { write } from './dom.js';
 import { enableToggles } from './ui.js';
 import { insertImage } from './capi-images.js';
 import { setEditionLink } from './ads';
+import { URLSearchParams } from './utils';
```
Comment (janua) [inline src/_shared/js/capi-multiple.js]: You aren't using this import?

Comment (janua) [inline src/_shared/js/capi-multiple.js]: You are! :clap:

**Comment by janua** [comment]
:+1:

**Comment by regiskuckaertz** [comment]
[Native templates: polyfills](https://trello.com/c/mIV1jlLK/91-native-templates-polyfills)

**Pull request merged**